build/Dockerfile

```diff
@@ -12,38 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi9/python-311 as wheel
 
-ARG WHEEL_VERSION=""
-USER root
-RUN --mount=type=cache,target=/root/.cache/pip \
-    python -m pip install --upgrade pip && \
-    python -m pip install build
-COPY tuning tuning
-COPY .git .git
-COPY pyproject.toml pyproject.toml
-# build wheel if wheel version is empty else download the wheel from PyPi
-RUN if [[ -z "${WHEEL_VERSION}" ]]; \
-    then python -m build --wheel --outdir /tmp; \
-    else pip download fms-hf-tuning==${WHEEL_VERSION} --dest /tmp --only-binary=:all: --no-deps; \
-    fi && \
-    ls /tmp/*.whl >/tmp/bdist_name
+## Global Args #################################################################
+ARG BASE_UBI_IMAGE_TAG=9.3-1552
```

Comment:
Do we want to set the base image to such a defined level? If we just tag with `9.3` we can get patch updates. But they have already released 9.4 and currently we are taking latest. So this means we would have to actively update the base image versus picking up the new one automatically